redis/client.py

```diff
@@ -1612,6 +1612,24 @@ def bitpos(self, key, bit, start=None, end=None):
                             "when end is specified")
         return self.execute_command('BITPOS', *params)
 
+    def copy(self, source, destination, destination_db=None, replace=False):
+        """
+        Copy the value stored in the ``source`` key to the ``destination`` key.
+
+        ``destination_db`` an alternative destination database. By default, 
+        the ``destination`` key is created in the source Redis database.
+
+        ``replace`` whether the ``destination`` key should be removed before
+        copying the value to it. By default, the value is not copied if
+        the ``destination`` key already exists.
+        """
+        params = [source, destination]
+        if destination_db:
```

Comment:
Can you change if change to [PEP-8](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html)'s ```if destination_db is None```

Reply:
Sure, no problem